Mathlib/Data/Matrix/Basic.lean

```diff
@@ -613,6 +613,13 @@ theorem mapMatrix_trans (f : α ≃ₐ[R] β) (g : β ≃ₐ[R] γ) :
     f.mapMatrix.trans g.mapMatrix = ((f.trans g).mapMatrix : Matrix m m α ≃ₐ[R] _) :=
   rfl
 
+/-- For any algebra `α` over a ring `R`, we have an `R`-algebra isomorphism
+`Matₙₓₙ(αᵒᵖ) ≅ (Matₙₓₙ(R))ᵒᵖ` given by transpose. -/
```

Comment:
Could you add a cross-reference to `Matrix.transposeAlgEquiv`, which is the commutative version?